tools/src/h5repack/h5repack_filters.c

```diff
@@ -219,7 +217,6 @@ aux_assign_obj(const char  *name,        /* object name from traverse list */
         }
     }
 
-    *obj = tmp;
```

Comment:
Not a good idea to return a stack object through a pointer...

Reply:
Yeah, I tend to agree